source/_posts/2016-12-03-remote-websockets-sonarr.markdown

```diff
@@ -36,14 +36,13 @@ The support for multiple hosts is a result of allowing mixed configurations with
 
 This release includes a new [websockets][websockets] based API by [@balloob] to power the next generation of Home Assistant frontends. The current frontend has been partly migrated to use it and will be further migrated in the future.
 
-## All changes 
+## All changes
 
 - New services and improved device support for [HomeMatic][homematic] ([@pvizeli], [@danielperna84])
 - Device tracker: New support for [GPSLogger][gpslogger] ([@dainok])
 - Sensor: Support for [Sonarr][sonarr] ([@hborawski])
 - Sensor: [World Air Quality Index][waqi] sensor ([@valentinalexeev], [@fabaff])
 - Sensor: Support for [Dutch Smart Meter Requirements][dsmr] ([@aequitas])
-- Switch: [Hook][hook] support by hooksmarthome.com ([@dasos])
```

Comment:
I wonder if we should keep the changelog line, but just remove the link. What do you think Frenck?

Reply:
Yes, as I mentioned here: <https://github.com/home-assistant/home-assistant.io/pull/11832#discussion_r369098417>

Reply:
Good point ✌ done with https://github.com/home-assistant/home-assistant.io/pull/11832/commits/408340b8c9d676dda28b2c48b3efb26f7f428085